microsite/src/main/tut/Laws.md

```diff
@@ -0,0 +1,92 @@
+---
```

Comment:
You need to add this to the navigation via `menu.yml`, otherwise nobody can get to it :D

Reply:
Heh, good to know, will do.

How is scalaz.org kept up to date with Scalaz 8?

Reply:
<-- This guy :D